xbmc/addons/kodi-addon-dev-kit/UseMultiArch.cmake

```diff
@@ -0,0 +1,49 @@
+# - Multiarch support in object code library directories
```

Comment:
Not sure has used the already present files who used for add-ons with file from [kodi-platform](https://github.com/xbmc/kodi-platform/blob/master/UseMultiArch.cmake)